services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/SdtMessageTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.moj.sdt.services.messaging;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(MockitoExtension.class)
+public class SdtMessageTest {
+
+    private static final String TEST_REFERENCE = "Test_Reference";
+    @Test
+    void getSdtRequestReferenceTest(){
+        SdtMessage sdtMessage = new SdtMessage();
```

Comment:
Could create a class field for the SdtMessage object and initialise it in a setUp() method, annotated with BeforeEach.  This would save you having to do it in each test.

Reply:
done